examples/token-server.conf

```diff
@@ -13,10 +13,15 @@ api {
 }
 
 shibboleth {
+    # Ensure only trusted clients can send us authentication information.
     header_prefix = "Identity-"
     upstream = ["127.0.0.1/16", "::1/64", "192.168.0.0/24"]
-}
+    # Names of the headers containing authentication information.
+    header_user_id = "Identity-OAuthUser"
+    header_user_permissions = "Identity-OAuthUserScopes"
+ }
 
 ui {
     page_size = 20
+    elephant = "./static/"
```

Comment:
Yes, it does. I'll rebase and implement the things in here at some point, but not yet.